### ISSUE TITLE
Whenever a filter is set, I get "none required" as a response

### ISSUE THREAD

**Issue by sirmmo** [hbakri/django-ninja-crud#378]
Whenever I set a "filters" parameter in query, the response I get is 

```
{
  "type": "none_required",
  "loc": [
  "query",
  "filters"
  ],
  "msg": "Input should be None"
}
```

The viewset is set up as 
```

prouter = Router()
class ProviderViewset(viewsets.ModelViewSet):
    model = Provider
    default_input_schema = ProviderIn
    default_output_schema = ProviderOut
    filter_schema = ProviderFilter

    list_view = views.ListModelView()
    create_view = views.CreateModelView()
    retrieve_view = views.RetrieveModelView()
    update_view = views.UpdateModelView()
    delete_view = views.DeleteModelView()


ProviderViewset.register_routes(prouter)

router.add_router('/providers/', prouter)

```

and the ProviderFilter extends ninja's FilterSchema

**Comment by hbakri** [comment]
Hi @sirmmo, thanks for reporting this issue! The behaviour you're seeing is related to a known issue, already addressed in PR #361. This PR fixes a problem where an empty `FilterSchema` was incorrectly set as a default, leading to errors when the "filters" key was included in requests.

Regarding your specific case, why do you encounter this even though you defined a `filter_schema`? The issue arises because `filter_schema` is set at the wrong level. In `django-ninja-crud`, `filter_schema` should be defined at the `view` level (cf. [documentation](https://django-ninja-crud.readme.io/reference/list-model-view)), not at the `ModelViewSet` level. Here's the correct way to set it up:
```python
router = Router()

class ProviderViewSet(viewsets.ModelViewSet):
    model = Provider
    default_input_schema = ProviderIn
    default_output_schema = ProviderOut

    list_providers = views.ListModelView(filter_schema=ProviderFilter)
    create_provider = views.CreateModelView()
    retrieve_provider = views.RetrieveModelView()
    update_provider = views.UpdateModelView()
    delete_provider = views.DeleteModelView()

ProviderViewSet.register_routes(router)
```
This change should resolve your issue! 

Additionally, I've renamed the view attributes (like `list_view` to `list_providers`) in the code example above. While this doesn't affect the current version (v0.4.1), it's important for the upcoming `v0.5` release. In `v0.5`, the attribute name will be used as the `operation_id`, which directly influences the route summary in the generated OpenAPI documentation. This ensures more descriptive and meaningful documentation for your API routes.

**Comment by sirmmo** [comment]
Thank you, will try it later today 🙏
